mne_bids/read.py

```diff
@@ -761,7 +761,8 @@ def read_raw_bids(bids_path, extra_params=None, verbose=None):
 
 @verbose
 def get_head_mri_trans(bids_path, extra_params=None, t1_bids_path=None,
-                       fs_subject=None, fs_subjects_dir=None, verbose=None):
+                       fs_subject=None, fs_subjects_dir=None, kind="",
```

Comment:
can we make all future params keyword only?

```suggestion
                       fs_subject=None, fs_subjects_dir=None, *, kind="",
```